.github/workflows/build.yaml

```diff
@@ -11,6 +11,13 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Run GoReleaser dry run
+        uses: goreleaser/goreleaser-action@v2
+        with:
+          version: latest
+          args: --snapshot  --skip-publish --rm-dist 
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we need the token to do a dry run?

Reply:
Haytham suggested me....and I think it is a good idea. it will verify all pf of go releaser changes

Reply:
I don't think Yee is disagreeing, he's just saying we don't need the token to do a dry-run... which I think is true...
```suggestion
```